pkg/remote/aws_ec2_scanner_test.go

```diff
@@ -232,7 +232,9 @@ func TestEC2Eip(t *testing.T) {
 			test:    "no eips",
 			dirName: "aws_ec2_eip_empty",
 			mocks: func(repository *repository.MockEC2Repository, alerter *mocks.AlerterInterface) {
```

Comment:
This test is for empty result, may be we should create a new test instead? Or at least rename this one.

Reply:
I think it's OK because this is an empty case we're handling here, just another empty scenario we do not have considered.